src/scanning/ScanningOptionsScreen/ScanningOptionsScreen.jsx

```diff
@@ -0,0 +1,131 @@
+  } = useQuery({
+    queryKey: ["candidates"],
+    queryFn: () =>
+      searchCandidates({
+        token: userInfo.token,
+        instanceUrl: userInfo.instance.url,
+        platform: Capacitor.getPlatform(),
+      }),
+  });
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting },
+    reset,
+    getValues,
+  } = useForm();
+  const onSubmit = (data) => {
+    console.log(data);
+    history.push("/app/scanning/main");
+  };
+  const savedStatusSelectOptions = [
+    { value: "all", label: "regardless of saved status" },
+    { value: "savedToAllSelected", label: "saved to all selected groups" },
+    {
+      value: "savedToAnySelected",
+      label: "saved to at least one of the selected groups",
+    },
+    {
+      value: "savedToAnyAccessible",
+      label: "saved to at least one group I have access to",
+    },
+    {
+      value: "notSavedToAnyAccessible",
+      label: "not saved to any of group I have access to",
+    },
+    {
+      value: "notSavedToAnySelected",
+      label: "not saved to any of the selected groups",
+    },
+    {
+      value: "notSavedToAllSelected",
+      label: "not saved to all of the selected groups",
+    },
+  ];
+  return (
+    <IonPage>
+      <IonContent>
+        <IonHeader>
+          <IonToolbar>
+            <IonTitle>Scanning options</IonTitle>
+          </IonToolbar>
+        </IonHeader>
+        <div className="scanning-options-container">
+          <form
+            className="scanning-options-form"
+            onSubmit={handleSubmit(onSubmit)}
+          >
+            <section className="dates-section">
+              <label htmlFor="start-date">Start date:</label>
+              <IonDatetimeButton datetime="datetime"></IonDatetimeButton>
+              <label htmlFor="end-date">End date:</label>
+              <IonDatetimeButton datetime="datetime"></IonDatetimeButton>
+            </section>
+
+            <section className="filtering-section">
+              <IonModal keepContentsMounted={true}>
+                <IonDatetime id="datetime"></IonDatetime>
+              </IonModal>
+              <div className="saved-status-container">
+                <label htmlFor="saved-status">Show candidates...</label>
+                <IonSelect
+                  id="saved-status"
+                  label="Filter"
+                  placeholder="Select a filtering option"
+                >
+                  {savedStatusSelectOptions.map((option) => (
+                    <IonSelectOption key={option.value} value={option.value}>
+                      {option.label}
+                    </IonSelectOption>
+                  ))}
+                </IonSelect>
+              </div>
+              <IonToggle>Hide rejected</IonToggle>
+            </section>
+
+            <section className="project-section">
+              <IonSearchbar placeholder="Search project"></IonSearchbar>
+              <IonSelect
+                label="Junk group"
```

Comment:
Should be plural

Reply:
Would a user choose multiple junk groups? Then the discarded sources would be added to both? The users I met with were only scanning for a project and discarding to this project's junk

Reply:
I clearly have seen some folks pick more than one junk group at once in some instances, but indeed it doesn't seem to be the norm. I would just make it a multiple selectable dropdown or something so the feature just supports both by design.

Reply:
Now that I think of it, I have also seen people select multiple junk groups. It was because they wanted to have the "not added to any of the selected groups" and then selected junk groups from different overlapping projects. Would it not be better if they could select "include" groups and "exclude" groups? Then choose only one junk group? Otherwise what would be the behavior for the discard action? Should it discard to all junk groups or let the user choose what group to discard to?

Reply:
Let the user choose, but the main goal of junk group is tho couple it with "not added to any of the selected groups", to avoid scanning for what was saved to junk before. Saving to junk is not "that" common, from what I've seen most folks just ignore sources they don't care about unless they really show up too often. Currently the annoying part is that there is one "save" button that you have to click on and unselect the "real" groups to save to junk. If we do keep track of junk groups then it can be save / junk, and on select to both you would pick groups to save the candidate to. Keep in mind that the behavior is the same for junk and normal groups. You might scan for multiple groups at once and save candidates to a subset of these, just like you might save to a subset of the selected junk groups.

Truth is folks "hacked" their way around the rejection system by creating these junk groups, that's more a consequence of what felt more practical to them then what this "not added to any of the selected groups" was really intended for.

Reply:
Sadly there is no good or bad answer here. Some folks don't even use any junk groups. I would just - as you did - split the select in 2 distinct dropdowns (and we will flag the junk groups as such in the DB so you can split that list appropriately), and same for the save / junk buttons at the bottom of the screen. That will help with clarity and the UX in general, while keeping it simple and almost identical for both options. One nice thing you could do though, is not show the "junk" button at the bottom of the screen (or disable it, with a visual cue that it is disabled and a tooltip telling you why on hover, your choice) if no junk group(s) have been selected.

Reply:
Ok I referenced your messages in #21